rinja_derive/src/tests.rs

```diff
@@ -92,5 +138,25 @@ impl ::std::fmt::Display for Foo {{
         expr0 = &(&&::rinja::filters::AutoEscaper::new(&(s), ::rinja::filters::Text)).rinja_auto_escape()?,
     )?;
 }"#,
+        3,
+    );
+
+    // In this test we make sure that every used template gets referenced exactly once.
+    let path = Path::new(env!("CARGO_MANIFEST_DIR")).join("templates");
+    let path1 = path.join("include1.html");
+    let path2 = path.join("include2.html");
+    let path3 = path.join("include3.html");
+    compare(
+        r#"{% include "include1.html" %}"#,
+        &format!(
+            r#"const _: &[::core::primitive::u8] = ::core::include_bytes!({path1:#?});
+            const _: &[::core::primitive::u8] = ::core::include_bytes!({path2:#?});
+            const _: &[::core::primitive::u8] = ::core::include_bytes!({path3:#?});
+            writer.write_str("3")?;
+            writer.write_str("3")?;
+            writer.write_str("3")?;
+            writer.write_str("3")?;"#
```

Comment:
Will be joined into one call with <https://github.com/rinja-rs/rinja/pull/86/commits/3c08d0b99f55919fed821058bb9578b4f7eba7b7>.